docs/decisions/0005-bulk-enrollment.rst

```diff
@@ -0,0 +1,59 @@
+Bulk Enrollment endpoint for Subscription learners
+--------------------------------------------------
+
+Status
+======
+
+Accepted
+
+Context
+=======
+
+Business need: The admin portal feature to enroll multiple learners in multiple subscription-courses
+needs a suitable backend.
+
+Various enrollment apis exist, but they do not cater to or optimize for multiple enrollments.
```

Comment:
suggestion: You may wish to cross check this part with the (not publicly available so can't just link to it) Miro board that covers our enrollment paths, just to see if everything is covered https://miro.com/welcomeonboard/ce4L6lSmyZ1mMN8FX9XghAihnyk6nuBDGezLZ3AXnyJrAehZvUtpqbrq410uB0mO

Reply:
thanks! yes I went over that, looks like the paths mentioned there lead to enrolling via the GrantDatasharingPermissions view which calls one of the methods I listed here, this is good to see though!